hardhat.config.ts

```diff
@@ -46,7 +46,19 @@ const config: HardhatUserConfig = {
         version: "0.8.2",
         settings: {
           evmVersion: "istanbul",
-          optimizer: { enabled: true, runs: 200 },
+          optimizer: { enabled: true, runs: 2000 },
+          outputSelection: {
+            "*": {
+              "*": ["storageLayout"],
```

Comment:
what does it mean?

Reply:
It generates a map of the storage layout used by members of a contract. Used by openzeppelin plugin to verify upgrade compatibility.